test/files/run/dotty-i11332b.scala

```diff
@@ -0,0 +1,20 @@
+// javaVersion: 11+
+// scalac: -release:11
+
+import java.lang.invoke._, MethodType.methodType
+
+object Test {
+  def main(args: Array[String]): Unit = {
+    val l = MethodHandles.lookup()
+    val mhCL = l.findStatic(classOf[ClassLoader], "getPlatformClassLoader", methodType(classOf[ClassLoader]))
+    // sanity check that the non-signature-polymorphic invocations work as expected
+    assert(null != (mhCL.invoke(): ClassLoader))
```

Comment:
`invoke` is also signature-polymorphic.  But it has different runtime semantics (it's looser than invokeExact) and it was what was actually used by `ZipArchiveTest` where it looks for `getPlatformClassLoader`, so I thought it would be a good variant.

Reply:
Thanks for catching my mistake here. I'll correct it before closing https://github.com/scala/bug/issues/12678